core/storage/collection/gae_models.py

```diff
@@ -32,12 +32,24 @@
 
 class CollectionSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
     """Storage model for the metadata for a collection snapshot."""
-    pass
+
+    @staticmethod
+    def get_export_policy():
+        """This model's export_data function has not been implemented yet.
```

Comment:
It sounds like there should be a TODO here. So, here and elsewhere, please change to something along the lines of:

```
       """This model's export_data function implementation is still pending.

       TODO(#8523): Implement this function.
       """
```

In addition, in #8523, please add a checkbox list of remaining models to fix (similar to how it's done in #4057) so that we can use that to track progress.

Reply:
Done.